pom.xml

```diff
@@ -218,7 +218,7 @@
         <version.org.wildfly.build-tools>1.1.3.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.core>3.0.0.Alpha5</version.org.wildfly.core>
-        <version.org.wildfly.arquillian>1.0.2.Final</version.org.wildfly.arquillian>
+        <version.org.wildfly.arquillian>1.1.0.Final</version.org.wildfly.arquillian>
```

Comment:
We should really move to 2.0.0.Final at this point.
Could you update PR to that.